db/connection.js

```diff
@@ -8,5 +8,8 @@ require('dotenv').config({
 if (!process.env.PGDATABASE) {
   throw new Error('PGDATABASE not set');
 }
+if(process.env.PGDATABASE){
```

Comment:
We don't need to check the PGDATABASE variable immediately after checking whether it doesn't exist. We can check that it doesn't exist and throw an error accordingly. 

If it does exist, then we would never enter that conditional logic block and we would carry on to export our pool of connections to the database

Reply:
okay, well we can delete that second if statement as we will have already checked for the variable existence in the previous conditional statement